liborbum/src/Controller/Ee/Vpu/Vu/Interpreter/CVuInterpreter_FLOAT.cpp

```diff
@@ -415,42 +811,116 @@ void CVuInterpreter::MULAbc_3(VuUnit_Base* unit, const VuInstruction inst)
 
 void CVuInterpreter::MADD(VuUnit_Base* unit, const VuInstruction inst)
 {
-    // TODO: Implement.
-#if defined(BUILD_DEBUG)
-    BOOST_LOG(Core::get_logger()) << boost::format("(%s, %d) MADD: Not implemented.") % __FILENAME__ % __LINE__;
-#else
-    throw std::runtime_error("MADD: Not implemented.");
-#endif
+    SizedQwordRegister& reg_source_1 = unit->vf[inst.fs()];
```

Comment:
ok

Reply:
turns out that there's an additional check for the multiplication part of `c + a * b`...
though the manual do seem to make the whole thing look complicated, so I'm not entirely sure that I handled the exceptions correctly